### PR TITLE
Restart services on unresponsive http

### LINKED ISSUE #8: Escualo should restart services if they get unresponsive though http

### PR DESCRIPTION
**Pull request by faloi**
Closes #8

### CODE REVIEW

**Comment by faloi** [comment]
@flbulgarelli please review this one.

File: bootstrap [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 HOST=$1
+MONIT_PASSWORD=$2
 
+MONIT_VERSION=5.16
 CONNECT=root@$HOST
 USE_SWAP=false
```
Comment (flbulgarelli) [inline bootstrap]: Don't forget to increment versions

**Comment by flbulgarelli** [comment]
Slowly, bootstrap and service scripts are becoming huge, and complex do to interpolations.

I think we should change the architecture of those scripts in the near future, so that instead of running a local script that executes a big script though ssh, we should just introduce a little script in the server that downloads escualo code from github, and executes it. 

That would allow us to easily split it into several files

**Comment by flbulgarelli** [comment]
Having said that, it looks ok for me

**Comment by faloi** [comment]
> I think we should change the architecture of those scripts in the near future, so that instead of running a local script that executes a big script though ssh, we should just introduce a little script in the server that downloads escualo code from github, and executes it.

I agree. I'd also like to improve the CLI, and I think migrating to Ruby could help (can you provide the link to the gem you mentioned the other day?).

**Comment by flbulgarelli** [comment]
It is https://github.com/erikhuda/thor

But I am not very convinced we should spent too much time on that, esualo CLI is quite raw but it works.

**Comment by faloi** [comment]
The problem underneath is that writing Bash is quite awful, and I find myself repeating code all the time. I really think the productivity -at least mine- would be better using Ruby.

I've also found https://github.com/commander-rb/commander, the one that `rhc` uses. Seems very easy to use.

**Pull request merged**